### ISSUE TITLE
Failed build caused by `export class ValidationError extends JoiError {`

### ISSUE THREAD

**Issue by breefstudio** [AndrewKeig/express-validation#135]
latest change in v3.0.7 break my build

incompatible type ValidationError and JoiError, error occured after remove package-lock.json and node_modules then run `npm i`

```
node_modules/express-validation/lib/index.d.ts:39:3 - error TS2416: Property 'name' in type 'ValidationError' is not assignable to the same property in base type 'ValidationError'.
  Type 'string' is not assignable to type '"ValidationError"'.

39   name: string;
     ~~~~

node_modules/express-validation/lib/index.d.ts:43:3 - error TS2416: Property 'details' in type 'ValidationError' is not assignable to the same property in base type 'ValidationError'.
  Type 'errors' is missing the following properties from type 'ValidationErrorItem[]': length, pop, push, concat, and 28 more.

43   details: errors;
     ~~~~~~~
```

**Comment by rickrille** [comment]
Hi, I got the same error today. Did you find a workaround?

**Comment by AndrewKeig** [comment]
sorry rolling back try 3.0.8

**Comment by rickrille** [comment]
Thanks, problem solved in 3.0.8
